### ISSUE TITLE
Placing tablist after tab panels causes twice modification warning 

### ISSUE THREAD

**Issue by Kilowhisky** [IvyApp/ivy-tabs#25]
If you place the tab list after the tab panels like so 

```
{{#ivy-tabs selection=fpSelection as |tabs|}}
    {{#tabs.tabpanel "tab1"}}
       AA
    {{/tabs.tabpanel}}
    {{#tabs.tabpanel "tab2"}}
       BB
     {{/tabs.tabpanel}}
     {{#tabs.tablist as |tablist|}}
          {{#tablist.tab "tab1" on-select=(action (mut fpSelection)) }}1{{/tablist.tab}}
          {{#tablist.tab "tab2" on-select=(action (mut fpSelection)) }}2{{/tablist.tab}}
      {{/tabs.tablist}}
 {{/ivy-tabs}}
```

Several errors are generated when tabbing that have to do with ember getting mad about twice modfying properties. 

```
DEPRECATION: You modified aria-hidden twice in a single render. This was unreliable in Ember 1.x and will be removed in Ember 3.0 [deprecation id: ember-views.render-double-modify]
        at logDeprecationStackTrace (http://localhost:4200/app/assets/vendor.js:16707:19)
        at HANDLERS.(anonymous function) (http://localhost:4200/app/assets/vendor.js:16817:7)
        at raiseOnDeprecation (http://localhost:4200/app/assets/vendor.js:16737:12)
        at HANDLERS.(anonymous function) (http://localhost:4200/app/assets/vendor.js:16817:7)
        at invoke (http://localhost:4200/app/assets/vendor.js:16833:7)
        at deprecate (http://localhost:4200/app/assets/vendor.js:16801:32)
        at Object.deprecate (http://localhost:4200/app/assets/vendor.js:29674:37)
        at Class.exports.default._emberMetalMixin.Mixin.create._Mixin$create.scheduleRevalidate (http://localhost:4200/app/assets/vendor.js:54153:26)
```

```
DEPRECATION: You modified (-join-classes "ember-view" "ivy-tabs-tabpanel" (-normalize-class "active" active activeClass=undefined inactiveClass=undefined)) twice in a single render. This was unreliable in Ember 1.x and will be removed in Ember 3.0 [deprecation id: ember-views.render-double-modify]
        at logDeprecationStackTrace (http://localhost:4200/app/assets/vendor.js:16707:19)
        at HANDLERS.(anonymous function) (http://localhost:4200/app/assets/vendor.js:16817:7)
        at raiseOnDeprecation (http://localhost:4200/app/assets/vendor.js:16737:12)
        at HANDLERS.(anonymous function) (http://localhost:4200/app/assets/vendor.js:16817:7)
        at invoke (http://localhost:4200/app/assets/vendor.js:16833:7)
        at deprecate (http://localhost:4200/app/assets/vendor.js:16801:32)
        at Object.deprecate (http://localhost:4200/app/assets/vendor.js:29674:37)
        at Class.exports.default._emberMetalMixin.Mixin.create._Mixin$create.scheduleRevalidate (http://localhost:4200/app/assets/vendor.js:54153:26)
        at http://localhost:4200/app/assets/vendor.js:27925:32
```

This was not an issue in the last version of ivy-tabs

**Comment by nbibler** [comment]
Thanks for the report. We looked into it at the time you mentioned it and haven't found a great (simple) fix, yet. We're going to keep working on it, though.

**Comment by omghax** [comment]
This has been fixed and released as version 3.0.1.
